packages/compass-components/package.json

```diff
@@ -73,7 +73,7 @@
     "@react-stately/tooltip": "^3.0.5",
     "@types/lodash": "^4.14.172",
     "bson": "^4.6.1",
-    "focus-trap-react": "^9.0.2",
+    "focus-trap-react": "8.4.2",
```

Comment:
I asked lg team why do they have it as an exact dep instead of a range. There doesn't seem to be a reason and managing exact dep might be a maintenance pain the future (especially because they bumped it at some point, but still with no explanation why it's an exact dep). Can we make it a caret range on our side? This way npm should be able to resolve it correctly and still have the same exact version as leafygreen expects, but when we update their components in the future it should auto resolve their expectation correctly

Reply:
Thanks for asking in their channel, updating to a caret for future compat.